apps/storefront/src/components/table/B3Table.tsx

```diff
@@ -355,7 +355,6 @@ export function B3Table<Row>({
               count={count}
               rowsPerPage={first}
               page={first === 0 ? 0 : offset / first}
-              of={b3Lang('global.pagination.of')}
```

Comment:
These are not part of the API for MUI, TS is complaining about them.

Will reintroduce properly if needed in the future.